Updates to RKG ceremony to caveat publication of internal documents

LewisA-r3 opened the pull request:
Original RKG ceremony script & annotated script were classified and marked as "internal". A caveat has been added to the title page, as it is inappropriate to modify the documents themselves.

LewisA-r3 committed:
Updates to RKG ceremony to caveat publication of internal documents

Original RKG ceremony script & annotated script were classified and marked as "internal". A caveat has been added to the title page, as it is inappropriate to modify the documents themselves.

KadriSiff reviewed (approved):
Hi Lewis - looking at the changes, I think all should display nicely!

LewisA-r3 committed:
pdfs move to asset directory

The amended PDF files moved to the asset/documents directory. urls updated accordingly. index.md file in the root directory amended - this is the only effective index.md file.

trust-root/index.md is not a functional component of the website - nonetheless, updated that for the aesthetics.
